website/integrations/services/hashicorp-vault/index.md

```diff
@@ -87,6 +87,8 @@ vault write auth/oidc/role/reader \
       user_claim="sub" \
       policies="reader"
 ```
-
+:::note
+If you intend to create [external groups](https://developer.hashicorp.com/vault/tutorials/auth-methods/oidc-auth#create-an-external-vault-group) in vault to manage user access the OIDC role will need to specifically request a custom scope using the `oidc_scopes` option when writing the OIDC role.
```

Comment:
```suggestion
If you intend to create [external groups](https://developer.hashicorp.com/vault/tutorials/auth-methods/oidc-auth#create-an-external-vault-group) in Vault to manage user access the OIDC role will need to specifically request a custom scope using the `oidc_scopes` option when writing the OIDC role.
```

Reply:
Instead of "writing the OIDC role" would it work to say "when defining..." or "when creating..."?

Reply:
Vault says "create" in their docs, so we probably should go for that

Reply:
I’ve made the change including capitalisation and using the same terminology (create) as the Hashicorp docs, good spot Risson